vault/logical_system_paths.go

```diff
@@ -1217,7 +1367,43 @@ func (b *SystemBackend) leasePaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
 					Callback: b.handleLeaseLookup,
-					Summary:  "Retrieve lease metadata.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"id": {
+									Type:        framework.TypeString,
+									Description: "Lease id",
+									Required:    true,
+								},
+								"issue_time": {
+									Type:        framework.TypeTime,
+									Description: "Timestamp for the lease's issue time",
+									Required:    true,
+								},
+								"renewable": {
+									Type:        framework.TypeBool,
+									Description: "True if the lease is able to be renewed",
+									Required:    true,
+								},
+								"expire_time": {
+									Type:        framework.TypeTime,
+									Description: "Optional lease expiry time ",
+									Required:    false,
```

Comment:
Yeah but given goes zero value for time if the key is set something will always been sent, so this may actually even be a required field? @antonakv do you have any further insights here?

Reply:
.... oops

Reply:
```suggestion
									Required:    true,
```